src/seeders/20180704134410-demo-admin.js

```diff
@@ -5,18 +5,19 @@ const hashFactory = require('../helpers/hash/hash-factory')();
 
 module.exports = {
   // eslint-disable-next-line no-unused-vars, no-use-before-define
-  async up(queryInterface, Sequelize) { return queryInterface.bulkInsert('admins', generate50Admins(), {}); },
+  async up(queryInterface, Sequelize) { return queryInterface.bulkInsert('admins', await generate50Admins(), {}); },
   // eslint-disable-next-line no-unused-vars
   down(queryInterface, Sequelize) { return queryInterface.bulkDelete('admins', null, {}); },
 };
 
 async function generate50Admins() {
   const admins = [];
+  const hash = await hashFactory.encrypt('admin');
   admins.push({
     first_name: 'admin',
     last_name: 'admin',
     email: 'admin@moodle.com',
-    password: await hashFactory.encrypt('admin'), // 'admin' - original password
+    password: await hash.toString('hex'), // 'admin' - original password
     phone_number: '000-581-5483',
```

Comment:
a better approach will be that the encrypt function will return a string :)

Reply:
this was the only thing that worked, the function do not want to return a string 
it's not transformig the string into "hex"